dashboard/app/models/sections/section.rb

```diff
@@ -341,4 +341,14 @@ def student_script_ids
   def unused_random_code
     CodeGeneration.random_unique_code length: 6, model: Section
   end
+
+  # Drops unicode characters not supported by utf8mb3 strings (most commonly emoji)
+  # from the section name.
+  # We make a best-effort to make the name usable without the removed characters.
+  # We can remove this once our database has utf8mb4 support everywhere.
+  def strip_emoji_from_name
+    self.name = name&.gsub(/[\u{10000}-\u{10FFFF}]/, '')&.strip
```

Comment:
Should we unify on an emoji-stripping technique in Ruby?  We use a different one here: https://github.com/code-dot-org/code-dot-org/pull/13030/files#diff-5db5628667c46ffae763784b229e77c1R59

Reply:
:man_facepalming: Yep, I should definitely have done it that way.  Thanks!